subsys/settings/src/settings_priv.h

```diff
@@ -22,13 +22,6 @@ struct mgmt_cbuf;
 int settings_cbor_line(struct mgmt_cbuf *cb, char *name, int nlen, char *value,
 		       int vlen);
 
-int settings_line_parse(char *buf, char **namep, char **valp);
-int settings_line_make(char *dst, int dlen, const char *name, const char *val);
-int settings_line_make2(char *dst, int dlen, const char *name,
```

Comment:
What about this one? You didn't remove it from the code?

Reply:
^^ was even not implemented.

Reply:
@carlescufi ^^